modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -39,78 +46,130 @@ rollback more easily if necessary.**
-```
-
-## General Changes
-
-### Remove Default Imports
+### Removal of Default Exports
 
-As part of some cleanup, we've removed any default imports that where lingering. This change matches
-more closely what we've been doing with new components and systems. As part of this change, the
-components listed below have moved from default imports to named imports.
+We've removed all remaining default exports from our packages to maintain consistency with our newer
+(and recently updated) components and systems which favor named exports. The following components
+have moved from default exports to named exports. The following components have been updated to
+provide named exports instead of default exports (in cases where the component provided both default
+and named exports, the default export has been removed).
 
-##### Fonts
+#### Main (canvas-kit-react)
 
-- fonts
-
-##### Main Package Components
-
-- Toast
-- Canvas
-- Tooltip
-- TextArea
+- AccentIcon
+- AppletIcon
 - Avatar
-- CountBadge
+- Canvas
 - Checkbox
+- CountBadge
 - FormField
+- Graphic
+- Icon
 - LoadingDots
 - Radio
 - SegmentedControl
 - Select
 - SidePanel
 - StatusIndicator
 - Switch
+- Svg
+- SystemIcon
+- SystemIconCircle
 - Table
+- TextArea
 - TextInput
+- Toast
+- Tooltip
 
-##### Labs Components
+#### Labs (canvas-kit-labs-react)
 
 - Combobox
 - Drawer
 
-##### Preview Components
+#### Preview (canvas-kit-preview-react)
 
 - Breadcrumbs
 - ColorPicker
 - Menu
 - Select
 - SidePanel
 
-🤖 The codemod will update imports to be named imports
+#### Fonts (canvas-kit-react-fonts)
+
+- fonts
+
+🤖 The codemod will rewrite your default imports to named imports.
+
+### Responsive Styles
+
+> (PLACEHOLDER) Responsive styles go here (https://github.com/Workday/canvas-kit/issues/1503)
+
+## New Components
+
+### Grid
+
+> (PLACEHOLDER) Introduction to Grid plus link to Grid docs goes here.
+
+### Text
+
+> (PLACEHOLDER) Introduction to Text plus link to Text docs goes here.
+
+## Component Deprecations
+
+### Deprecation Types
+
+There are two types of deprecations: soft and hard.
+
+#### Soft Deprecation
+
+A soft-deprecated component is still available with its full functionality, but it will have been
+renamed with a prefix to indicate its soft-deprecated status. It will also include a console warning
+announcing its deprecation. This warning will only be triggered on the component's initial render.
+
+Soft-deprecated types and utilities will also be renamed but generally will not trigger a console
+warning.
+
+#### Hard Deprecation
+
+A hard-deprecated component or package is no longer available. You will need to follow the method
+prescribed in our upgrade guide to update your application. Please
+[reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) to our team
+directly if you need additional help.
+
+> (PLACEHOLDER) Soft deprecation of Drawer, Menu (Preview), Layout and Column goes here. (https://github.com/Workday/canvas-kit/pull/1747)
```

Comment:
Hey James, what are your thoughts on guiding consumers to use our main package components if were' soft deprecating? Example, menu in preview is being deprecated, should we guide them to use the main package one?

Reply:
Yes, that is a great idea -- we should definitely guide consumers towards using the Menu in Main 👍